.github/workflows/callable-prebuild-samples-staging.yml

```
@@ -0,0 +1,20 @@
+name: Prebuild AmplifyJs
```

```suggestion
name: Prebuild Amplify Integration Test Package
```

(or Samples Staging)

---

On it.